sdk/communication/Azure.Communication.JobRouter/src/Models/UpdateWorkerOptions.cs

```diff
@@ -29,13 +29,13 @@ public UpdateWorkerOptions(string workerId)
         /// <summary>
         /// Unique key that identifies this worker.
         /// </summary>
-        public string WorkerId { get; set; }
+        public string WorkerId { get; }
 
         /// <summary> The total capacity score this worker has to manage multiple concurrent jobs. </summary>
-        public int TotalCapacity { get; set; }
+        public int? TotalCapacity { get; set; }
 
         /// <summary> A flag indicating this worker is open to receive offers or not. </summary>
-        public bool AvailableForOffers { get; set; }
+        public bool? AvailableForOffers { get; set; }
```

Comment:
Same here.

Reply:
Maybe a generic wrapper of `Optional<T>` needs to be created.

Reply:
Same here